LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -1696,12 +1698,26 @@ void LogFileMonitor::WriteToConsole( _In_ std::wstring Message, _In_ std::wstrin
         if (msg.size() > 0) {
             // escape backslashes in FileName
             auto fmtFileName = Utility::ReplaceAll(FileName, L"\\", L"\\\\");
-            // sanitize msg
-            Utility::SanitizeJson(msg);
-            auto log = Utility::FormatString(logFmt, msg.c_str(), fmtFileName.c_str());
-            logWriter.WriteConsoleLog(log);
-        }
+            std::wstring formattedlog, logFmt;
+            if (Utility::CompareWStrings(m_logFormat, L"JSON"))
+            {
+                logFmt = L"{\"Source\": \"File\",\"LogEntry\": {\"Logline\": \"%s\",\"FileName\": \"%s\"},\"SchemaVersion\":\"1.0.0\"}";
+                // sanitize message
+                Utility::SanitizeJson(msg);
+
+                formattedlog = Utility::FormatString(

```

Comment:
nit: this is repeated in both parts of the if/else, could just move it out of the if/else.

Reply:
Addressed